### PR TITLE
FE-652 | Add ContainsField

### PR DESCRIPTION
**Pull request by fauna-ryanharris**
### Notes
[Jira Ticket](https://faunadb.atlassian.net/browse/FE-652)

This PR adds the `ContainsField` function to our JavaScript driver. I have also included some test coverage that reflects how the [functionality is defined within Core](https://github.com/fauna/core/blob/09784957af4b33f597761ac768283f7a8eb6edbf/ext/api/src/test/scala/queries/BasicFunctionsSpec.scala#L1214).

The function takes two arguments:
1. A string representing the field to search for (rejects arrays, objects and numbers)
2. An object which we'll search its root for the field value

### CODE REVIEW

**Comment by BrunoQuaresma** [comment]
@ryancharris tests are broken in all environments 😕

File: test/query.test.js [hunk 1]
```diff
@@ -1906,17 +1936,19 @@ describe('query', () => {
   test('and', () => {
     var p1 = assertQuery(query.And(true, true, false), false)
     var p2 = assertQuery(query.And(true, true, true), true)
-    var p3 = assertQuery(query.And(true), true)
-    var p4 = assertQuery(query.And(false), false)
-    return Promise.all([p1, p2, p3, p4])
+    // var p3 = assertQuery(query.And(true), true)
```
Comment (fauna-ryanharris) [inline test/query.test.js:1939]: These are commented out for now but will be re-added in the future: https://github.com/fauna/faunadb-js/pull/295

File: src/query.js [hunk 2]
```diff
@@ -1956,6 +1956,20 @@ function Contains(path, _in) {
   return new Expr({ contains: wrap(path), in: wrap(_in) })
 }
 
+/**
+ * See the [docs](https://app.fauna.com/documentation/reference/queryapi#miscellaneous-functions).
+ *
+ * @param {string} field
+ *   A value representing a field we want to confirm exists.
```
Comment (nathaniel-may) [inline src/query.js:1963]:
```suggestion
 *   A field name we want to confirm exists.
```

**Comment by BrunoQuaresma** [comment]
A late note, @ryancharris also check the Typescript types for the new, updated functions!

**Comment by fauna-ryanharris** [comment]
> A late note, @ryancharris also check the Typescript types for the new, updated functions! This is something we are missing sometimes.

@BrunoQuaresma That was originally part of the diff 🤔 Lemme see why it's not there anymore.

**Pull request merged**